app/views/templates/Layout.scala.html

```diff
@@ -52,6 +52,10 @@
   <link href='@controllers.routes.Assets.versioned("stylesheets/print.css")' media="all" rel="stylesheet" type="text/css" />
 }
 
+@customTechnicalProblemLinkSection = {
+    <p class="govuk-body"><a target="_blank" href=@messages("site.technicalProblemUrl") class="govuk-link">@messages("site.technicalProblemMessage")</a></p>
```

Comment:
If you implemented like suggested in `conf/messages.en` you can do the following below - 

```suggestion
    <p class="govuk-body"> @Html(Messages("site.technicalProblemMessage", routes.WhateverTheCurrentPageIs.view.url)) </p>
```